eng/codecov.yml

```diff
@@ -0,0 +1,11 @@
+codecov:
+  branch: master
+  ci:
+    - dnceng.visualstudio.com
+
+coverage:
+  range: 60...90
```

Comment:
❔ Can you describe the motivation behind this range? I always recommend that teams using code coverage tools _never_ gate on a particular coverage number, but I can think of other reasons why this might be included. I'm curious why it's used here.

Reply:
I lowered the lower bound temporarily as our coverage numbers aren't great for most of our libraries. As soon as codecov integration is enabled in corefx I will adjust these settings based on what makes sense.